### PR TITLE
Add frame index as argument

### PR DESCRIPTION
**Pull request by EnricaSo**
- add frame_index as an argument to the extrinsic calibration file: useful whenever the camera moves in the beginning
- fix path of the output directory

### CODE REVIEW

**Comment by EnricaSo** [comment]
Sorry, I pushed the fix for the plotter window and the path in this branch. I can remove them quickly if not needed.

File: extrinsic_calibration.py [hunk 1]
```diff
@@ -140,9 +141,16 @@ def save_extrinsic_calib(output_dir, rvec, tvec):
...
     cap = cv2.VideoCapture(args.video_file)
+
+    if args.frame_index is not None:
+        frame_index = int(args.frame_index)
+    else:
+        frame_index = 0
     ret = False
-    while not ret:
+    counter = 0
+    while not (ret and counter >= frame_index):
```
Comment (MauroPfister) [inline extrinsic_calibration.py]: I think the `while` loop is not required here. Instead use
```
cap.set(cv2.CAP_PROP_POS_FRAMES, frame_index)
ret, frame = cap.read()
```

**Comment by MauroPfister** [comment]
And yes, I prefer if you reverted the fix for the plotter window and output path. Adding `cv2.waitKey(0)` spawns a separate thread that is not terminated when the window is closed by clicking on the "cross" in the window corner. That is the reason for the strange `while` loop. I would rather find the reason why this is not working on your system.

**Comment by EnricaSo** [comment]
I've undone changes and the git history has diverged. I'll close this PR and create a new one.

**Pull request closed without merging**